MergeRequestTable.elm

```diff
@@ -31,17 +27,21 @@ type alias Model =
   , lastRefreshTime : Time
   }
 
+-- SUBSCRIPTIONS
+subscriptions: Model -> Sub Msg
+subscriptions model =
+  Time.every 5000 Tick
 
--- UPDATE
 
-type Action =
-  UpdateList (List MergeRequestStats)
+-- UPDATE
+type Msg
+  = UpdateList (Result Http.Error (List MergeRequestStats))
```

Comment:
I don't like this change. Before it was `Http`-free and seemed more appropriate because concerns of getting the MRs from the server and displaying them are separate. I think we should get rid of `Http.Error` before sending `UpdateList`.